src/stirling/source_connectors/socket_tracer/conn_tracker.h

```diff
@@ -799,6 +806,8 @@ std::string DebugString(const ConnTracker& c, std::string_view prefix) {
   info += absl::Substitute("state=$0\n", magic_enum::enum_name(c.state()));
   info += absl::Substitute("$0remote_addr=$1:$2\n", prefix, c.remote_endpoint().AddrStr(),
                            c.remote_endpoint().port());
+  info += absl::Substitute("$0remote_addr=$1:$2\n", prefix, c.local_endpoint().AddrStr(),
+                           c.local_endpoint().port());
```

Comment:
I think this was supposed to have the text `local_addr=$1:$2`.

Reply:
Fixed